src/Billing/BillingProcessor/BillingClaimBatch.php

```diff
@@ -20,8 +20,6 @@
 
 namespace OpenEMR\Billing\BillingProcessor;
 
-use OpenEMR\Billing\BillingProcessor\BillingClaimBatchControlNumber;
```

Comment:
hi @sjpadgett , great work getting this going again

pretty sure need this when using it below in lines 61 and 64

Reply:
nope it's already in scope

Reply:
oh, nvm, since it's in the same namespace :)

Reply:
Also a FYI the latest ci will optimize uses and removed unused automatically so be sure to keep an eye on and verify removals yourself.

Reply:
okay for phpstorm?